src/m365/teams/commands/meeting/meeting-create.ts

```diff
@@ -0,0 +1,206 @@
+        }
+        if (args.options.startTime && args.options.endTime && new Date(args.options.startTime) >= new Date(args.options.endTime)) {
+          return 'startTime value must be before endTime.';
+        }
+        if (args.options.endTime && !args.options.startTime) {
+          return 'startTime should be specified when endTime is specified.';
+        }
+        if (args.options.participants) {
+          if (args.options.participants.indexOf(',') === -1 && !validation.isValidUserPrincipalName(args.options.participants)) {
+            return `${args.options.participants} contains invalid UPN.`;
+          }
+          const participants = args.options.participants.trim().toLowerCase().split(',').filter(e => e && e !== '');
+          if (!participants || participants.length === 0 || participants.some(e => !validation.isValidUserPrincipalName(e))) {
+            return `${args.options.participants} contains one or more invalid UPN.`;
+          }
+        }
+        if (args.options.organizerEmail && !validation.isValidUserPrincipalName(args.options.organizerEmail)) {
+          return `'${args.options.organizerEmail}' is not a valid email for organizerEmail.`;
+        }
+        return true;
+      }
+    );
+  }
+
+  /**
+   * Executes the command
+   * @param logger Logger instance
+   * @param args Command arguments
+   */
+  public async commandAction(logger: Logger, args: CommandArgs): Promise<void> {
+    try {
+      const isAppOnlyAccessToken = accessToken.isAppOnlyAccessToken(auth.service.accessTokens[this.resource].accessToken)!;
+      if (isAppOnlyAccessToken && !args.options.organizerEmail) {
+        throw `The option 'organizerEmail' is required when creating a meeting using app only permissions`;
+      }
+      const graphBaseUrl = await this.getGraphBaseUrl(args.options);
+      const meeting = await this.createMeeting(logger, graphBaseUrl, args.options);
+      await logger.log(meeting);
+    }
+    catch (err: any) {
+      this.handleRejectedODataJsonPromise(err);
+    }
+  }
+
+  /**
+   * Gets the base MS Graph URL for the request
+   * @param options 
+   * @returns correct MS Graph URL for the request
+   */
+  private async getGraphBaseUrl(options: Options): Promise<string> {
+    let requestUrl = `${this.resource}/v1.0/`;
+    if (options.organizerEmail) {
+      const organizerId = await aadUser.getUserIdByEmail(options.organizerEmail);
+      requestUrl += `users/${organizerId}`;
+    }
+    else {
+      requestUrl += 'me';
+    }
+    return requestUrl;
+  }
+
+  /**
+   * Creates a new online meeting
+   * @param logger 
+   * @param graphBaseUrl 
+   * @param options 
+   * @returns MS Graph online meeting response
+   */
+  private async createMeeting(logger: Logger, graphBaseUrl: string, options: Options): Promise<any> {
+    if (this.verbose) {
+      logger.logToStderr(`Creation of a meeting...`);
+    }
+    const requestData: any = {};
+    if (options.participants) {
```

Comment:
We typically always add a line break above and below if-statements, for better readability

Reply:
Added :)